projects/core/src/util/index.ts

```diff
@@ -12,6 +12,7 @@ export * from './glob.service';
 export * from './http-params-uri.encoder';
 export * from './java-reg-exp-converter/index';
 export * from './normalize-http-error';
+export * from './object-comparison-utils';
```

Comment:
What do we need this import?

Reply:
to use the `ObjectComparisonUtils `in the `configurator-attribute-composition.directive.ts`